openapi.yaml

```diff
@@ -16,175 +16,511 @@ host: api.oregonstate.edu
+          schema:
+            $ref: '#/definitions/PlayerResults'
+        '404':
+          description: Game not found
+          schema:
+            $ref: '#/definitions/Errors'
+        '500':
+          description: Internal server error
+          schema:
+            $ref: '#/definitions/Errors'
+  /players/{playerID}:
+    get:
+      summary: Info for a specific player in a game
+      tags:
+        - players
+      description: get player by player id
+      operationId: getPlayerByID
+      parameters: 
+        - $ref: '#/parameters/authorization'
+        - name: playerID
+          in: path
+          description: g[int]p[int] where the first int is the ID of the game and the second int is the ID of the player in that game.
           required: true
           type: string
+          pattern: '^g[1-9]\d*p[1-9]\d*$'
       responses:
         '200':
           description: Successful response
           schema:
-            $ref: '#/definitions/PetResult'
+            $ref: '#/definitions/PlayerResult'
         '404':
-          description: PetResource not found
+          description: Game not found
+          schema:
+            $ref: '#/definitions/Errors'
+        '500':
+          description: Internal server error
+          schema:
+            $ref: '#/definitions/Errors'
+    put:
+      tags:
+        - players
+      summary: Update a player specified by its id
+      operationId: updatePlayerByID
+      consumes:
+        - application/json
+      produces:
+        - application/json
+      parameters: 
+        - name: playerID
+          in: path
+          description: ID of player to be updated
+          required: true
+          type: string
+          pattern: '^g[1-9]\d*p[1-9]\d*$'
+        - name: updateBody
+          in: body
+          description: Object that will overwrite the player specified by its id
+          required: true
+          schema:
+            $ref: '#/definitions/PlayerUpdateBody'
+      responses:
+        '200':
+          description: Successfully update a Player
+          schema:
+            $ref: '#/definitions/PlayerResult'
+          headers:
+            Location:
+              type: string
+              format: url
+              description: Location of the updated player
+        '400':
+          description: Invalid player object
           schema:
             $ref: '#/definitions/Errors'
         '500':
           description: Internal server error
           schema:
             $ref: '#/definitions/Errors'
+    delete:
+      tags:
+      - players
+      summary: "Remove a player specified by its id"
+      operationId: removePlayerByID
+      consumes:
+        - "application/json"
+      produces:
+        - "application/json"
+      parameters:
+        - name: playerID
+          in: "path"
+          description: "Id of player to be removed"
+          required: true
+          type: string
+          pattern: '^g[1-9]\d*p[1-9]\d*$'
```

Comment:
Since you've defined there can be [at most 5 players](https://github.com/osu-mist/poker-api/pull/1/files#diff-fe030a7c1568b3decf599edf399be7f4R459) in a game, will `g3p6` or `g3p10` be a valid playerId?